go/autogen/api/models.go

```diff
@@ -133,3 +133,47 @@ func (c *OllamaClientConfiguration) ToConfig() (map[string]interface{}, error) {
 func (c *OllamaClientConfiguration) FromConfig(config map[string]interface{}) error {
 	return fromConfig(c, config)
 }
+
+type BaseVertexAIConfig struct {
```

Comment:
Doesn't need to be right this minute, but I do think we should support machine credential source for `google`

Reply:
Let's open an issue and I can implement that as a followup

Reply:
opened https://github.com/kagent-dev/kagent/issues/477